### PR TITLE
Added custom bottom navigation

### PR DESCRIPTION
**Pull request by SantamRC**
Added a custom bottom navigation bar. 
The earlier Navigation Bar uses images for the background and icons which can create problems with the resolution. In this Navbar, the background is an uniform color and it uses fontawesome icons for best visual at different screen sizes.

Screenshot:

new navbar

### CODE REVIEW

**Comment by maedoc** [comment]
@liadomide should Michael review this?

File: framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html [hunk 1]
```diff
@@ -49,6 +49,8 @@
         <script type="text/javascript" src="{{ deploy_context | safe }}/static/jquery/jquery.json.js"></script>
         <script type="text/javascript" src="{{ deploy_context | safe }}/static/js/genericTVB.js?{{ currentVersionJS }}"></script>
 
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
```
Comment (liadomide) [inline framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html:53]: Please don't use external dependencies of this sort. Tvb web site needs to function also in a local context with poor or no connectivity

File: framework_tvb/tvb/interfaces/web/templates/jinja2/new_footer.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!--! ========================================================== -->
```
Comment (liadomide) [inline framework_tvb/tvb/interfaces/web/templates/jinja2/new_footer.html:1]: Ideally we would see changes in the prev footer.html file, for being able to compare code

**Review by liadomide — CHANGES_REQUESTED** [review]

File: framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html (hunk 1 above)
Comment (SantamRC) [reply framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html]: I have removed external dependencies in the latest commit and added the icons locally.

File: framework_tvb/tvb/interfaces/web/templates/jinja2/new_footer.html (hunk 2 above)
Comment (SantamRC) [reply framework_tvb/tvb/interfaces/web/templates/jinja2/new_footer.html]: Updated the footer.html file

**Comment by liadomide** [comment]
This is an interesting start. We hopefully take over in the future.
But I close this, as the GSOC project we planned to do the web changes wan't sponsored.

**Pull request closed without merging**